### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [notea-org/notea#159] [Q&A] [answered]
Hi.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by tecc** [comment, marked as answer]
Hi.
Whilst I wouldn't describe it as active (as in actively and regularly being worked on), it's not dead.
So yes, it is technically active. I recently released v0.4.0-alpha and I want to release a fully stable\* version hopefully early 2023, but I haven't gotten much work done as I haven't really had time for it.

**Reply by ghost** [reply]
Hi tecc, thank you for feedback.

**Reply by tecc** [reply]
Also, since you mentioned you wanted to help, in what way? I'm assuming help with code etc. but I'm just curious.
Of course, I'd greatly appreciate help with Notea, as would probably many of the users since that could potentially lead to more stuff getting done quicker.

**Reply by ghost** [reply]
Hi tecc.

> Also, since you mentioned you wanted to help, in what way? I'm assuming help with code etc. but I'm just curious.

ui/ux.
